tests/integration/defs/disaggregated/test_disaggregated.py

```
@@ -2165,11 +2252,33 @@ def test_disaggregated_gpt_oss_120b_harmony(disaggregated_test_root,
     pytest.param(TestConfig(model_path='gpt_oss/gpt-oss-120b',
                             test_desc='gpt_oss_120b_stress',
                             request_count=60000,
                             accuracy_threshold=0.42),
                  marks=(pytest.mark.skip_less_device(4), skip_pre_blackwell)),
+    pytest.param(TestConfig(
+        model_path='gpt_oss/gpt-oss-120b',
+        test_desc='gpt_oss_120b_eagle_stress',
+        request_count=60000,
+        accuracy_threshold=0.42,
+        speculative_model_path='gpt_oss/gpt-oss-120b-Eagle3'),
+                 marks=(pytest.mark.skip_less_device(8), skip_pre_hopper)),
```

Flake8 (7.3.0)

[error] continuation line over-indented for hanging indent

(E126)